src/App.tsx

```diff
@@ -1,11 +1,55 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { Filter } from './types/Filter';
+import { ErrorMessage } from './components/ErrorMessage/ErrorMessage';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
+  const [error, setError] = useState(false);
+  const [errorMessage] = useState('');
+
+  if (error) {
+    setTimeout(() => {
+      setError(false);
+    }, 3000);
+  }
```

Comment:
move this logic in `ErrorMessage`
better use useEffect, you need to create setTimeout on the first render and clear timeout on unmount